### PR TITLE
fix: Clarify crane install steps

### PR DESCRIPTION
**Pull request by okozachenko1203**

### CODE REVIEW

File: cmd/crane/README.md [hunk 1]
```diff
@@ -10,27 +10,40 @@ A collection of useful things you can do with `crane` is [here](recipes.md).
...
+3. Unpack it
+
+```sh
+$ tar -zxvf go-containerregistry.tar.gz
+```
+
+4. Find the crane binary in the unpacked directory, and move it to its desired destination
+
+```sh
+mv crane /usr/local/bin/crane)
```
Comment (imjasonh) [inline cmd/crane/README.md:40]:
```suggestion
mv crane /usr/local/bin/crane
```

File: cmd/crane/README.md [hunk 2]
```diff
@@ -10,27 +10,40 @@ A collection of useful things you can do with `crane` is [here](recipes.md).
...
   PASSED: Verified SLSA provenance
 ```
 
+3. Unpack it
+
+```sh
+$ tar -zxvf go-containerregistry.tar.gz
+```
+
+4. Find the crane binary in the unpacked directory, and move it to its desired destination
```
Comment (imjasonh) [inline cmd/crane/README.md:37]: I think we could make unpacking and moving one step:

```
tar xvfz go-containerregistry_${OS}_${ARCH}.tar.gz -C /usr/local/bin/ crane
```

**Comment by imjasonh** [comment]
You'll also need to sign the Google CLA.

**Comment by okozachenko1203** [comment]
> You'll also need to sign the Google CLA.

ok, signed

**Review by imjasonh — APPROVED** [review]

**Review by imjasonh — APPROVED** [review]

**Pull request merged**